scripts/scil_compute_fodf_metrics.py

```diff
@@ -3,8 +3,9 @@
 
 """
 Script to compute the maximum Apparent Fiber Density (AFD), the fiber ODFs
-orientations (peaks) and the Number of Fiber Orientations (NuFO) maps from
-fiber ODFs.
+orientations, values and indices (peaks, peak_values, peak_indices), the Number
+of Fiber Orientations (NuFO) maps from fiber ODFs, the RGB map, the Generalized
+Fractional Anisotropy (GFA) and the Quantitative Anisotropy (QA).
```

Comment:
Please remove GFA. It does not make sense to compute GFA from a fODF. GFA is a q-ball metric. Quantitative Anisotropy might be ok but I really don't know what this metric does... ? Isn't it a fixel-based metric as AFD? I thought it was the amplitude of the ODF.... So, is it QA_max? I would remove as well...